docs/reports/peer_testing_2/usability_tasks.md

```diff
@@ -67,9 +86,7 @@ Repeat this scenario with a different voice each time.
 
```

Comment:
here would the user add in voice effects on their text input from the Kukarella site ?

Reply:
yes, the user will input everything in Kukarella, that's correct